sunpy/net/hek/tests/test_utils.py

```diff
@@ -0,0 +1,175 @@
+import copy
+
+import numpy as np
+import pytest
+
+import astropy.table
+import astropy.time
+import astropy.units as u
+import astropy.utils
+import astropy.utils.masked
+from astropy.coordinates import SkyCoord
+from astropy.tests.helper import assert_quantity_allclose
+
+from sunpy.coordinates import (
+    Heliocentric,
+    HeliographicCarrington,
+    HeliographicStonyhurst,
+    Helioprojective,
+    get_earth,
+)
+from sunpy.net.hek.utils import (
+    _map_chain_code_columns_to_coordinates,
+    _map_columns_to_quantities,
+    _map_event_coord_columns_to_coordinates,
+    _parse_unit,
+)
+
+
+@pytest.mark.parametrize(('input_unit', 'expected_unit'), [
+    ('DN/sec/pixel', u.DN/(u.pix*u.s)),
+    ('ergs per cubic centimeter', u.erg/u.cm**3),
+    ('HMI pixel', 0.5*u.arcsec/u.pixel),
+    ('HMI pixels', 0.5*u.arcsec/u.pixel),
+    ('ma/m2', u.milliampere/u.m**2),
+    ('deg deg', u.deg),
+    ('arcsec, arcsec', u.arcsec),
+    ('arcsec,arcsec', u.arcsec),
+])
+def test_parse_unit(input_unit, expected_unit):
+    assert _parse_unit(input_unit) == expected_unit
+
+
+def map_event_coords_to_column(coord):
```

Comment:
can you give this a docstring or some explanatory comment because I don't know what it's trying to do.